signer/src/block_observer.rs

```diff
@@ -276,20 +278,50 @@ impl<C: Context, B> BlockObserver<C, B> {
     /// Process the bitcoin block. Also process all recent stacks blocks.
     #[tracing::instrument(skip_all, fields(block_hash = %block.block_hash()))]
     async fn process_bitcoin_block(&self, block: bitcoin::Block) -> Result<(), Error> {
+        let storage = self.context.get_storage_mut();
+        let bitcoin_client = self.context.get_bitcoin_client();
+        let stacks_client = self.context.get_stacks_client();
+
         tracing::info!("processing bitcoin block");
+
+        let until_bitcoin_height = stacks_client
```

Comment:
Hm, maybe we should just store this in the `SignerState` at startup since it should never change while running..

Reply:
Changed in [c7fb734](https://github.com/stacks-network/sbtc/pull/1087/commits/c7fb734e8b90856847a461c248e117146aa25ce9)